create-meta/javascript/src/createMeta.ts

```diff
@@ -0,0 +1,129 @@
+import os from 'os'
+import { messages, version as protocolVersion } from '@cucumber/messages'
+import defaultCiDict from './ciDict.json'
+
+export type CiDict = { [name: string]: CiSystem }
```

Comment:
shouldn't it be CIDict, CISystem etc ? (with a capital I ) - although it might make the variable names a bit weird later in the code :/

Reply:
I prefer it as is. My personal favorite: `XMLHTTPRequest`, reads much better when written as `XmlHttpRequest`.